basxconnect/mailer_integration/views.py

```diff
@@ -163,6 +128,26 @@ def display_previous_execution(request):
     )
 
 
+def display_sync_persons(sync_status):
+    return hg.F(
+        lambda c: hg.BaseElement(
+            *[
+                hg.BaseElement(
+                    hg.DIV(
+                        person.first_name,
+                        " ",
+                        person.last_name,
+                        " <",
+                        person.email,
+                        ">",
+                    )
+                )
+                for person in c["row"].persons.filter(sync_status=sync_status)
+            ]
+        )
+    )
+
+
```

Comment:
Just FYI, I imagine this could also be written somehow like this:

```python
    hg.Iterator(
        hg.F(lambda c: c["row"].persons.filter(sync_status=sync_status)),
        "person",
        hg.DIV(hg.format("{} {} <{}>", hg.C("person.first_name"), hg.C("person.last_name"), hg.C("person.email")))
    )

But it is a aestethic question, python vs the htmlgenerator-DSL
In any case, I would expect the angle-brackets to be escaped. Can you verify that this is happening?

Reply:
With `hg.format` it doesn't work properly...
![image](https://user-images.githubusercontent.com/11404846/143675539-e908b59f-7193-44c4-a87a-30bbc8ea13cf.png)

Is there a trick to display the angle-brackets correctly using `hg.format`?

Reply:
Okay, this is actually a security bug with ```hg.format```. It should escape the string properly, which it doesn't. I will fix that in htmlgenerator.

Reply:
Fixed it in htmlgenerator. Can you check again if this works with your code?

Reply:
works now, thanks!